data/Pate_1990/metadata.yml

```diff
@@ -0,0 +1,321 @@
+source:
+  primary:
+    key: Pate_1990
+    bibtype: Article
+    year: 1990
+    author: John S. Pate, Raymond H. Froend, Barbara J. Bowen, Annette Hansen, John
+      Kuo
+    title: Seedling growth and storage characteristics of seeder and resprouter species
+      of Mediterranean-type ecosystems of SW Australia
+    journal: Annals of Botany
+    volume: 65
+    number: 6
+    pages: 585--601
+    doi: 10.1093/oxfordjournals.aob.a087976
+contributors:
+  data_collectors:
+  - last_name: Pate
+    given_name: John
+    ORCID: .na
+    affiliation: School of Plant Biology, The University of Western Australia, Crawley,
+      Western Australia, Australia
+  austraits_curators: Sophie Yang
+dataset:
+  data_is_long_format: no
+  custom_R_code:                   '
+    data %>% 
+      mutate(
+        location = "Mediterranean-type ecosystems in south-western Australia",
+        shoot_root_ratio = 1/shoot_root_ratio,
```

Comment:
I've just renamed the new column root_shoot_ratio, because right now it suggests that this really is shoot_root_ratio, but there was a mistake in the input data

Reply:
Oh, just that if I saw this, I wouldn't be sure why the values were being flipped... Not a deep thought.